src/test/scala/org/broadinstitute/dsde/firecloud/utils/TSVFormatterSpec.scala

```diff
@@ -14,91 +15,104 @@ class TSVFormatterSpec extends FreeSpec with ScalaFutures with Matchers with Ins
+            "header_1" -> MockUtils.randomAlpha().toJson,
+            "header_2" -> MockUtils.randomAlpha().toJson,
+            "participant_id" -> """{"entityType":"participant","entityName":"participant_name"}""".parseJson
+          )
+        }
+        val sampleList = List(
+          EntityWithType("sample_01", "sample", Some(sampleAtts)),
+          EntityWithType("sample_02", "sample", Some(sampleAtts)),
+          EntityWithType("sample_03", "sample", Some(sampleAtts)),
+          EntityWithType("sample_04", "sample", Some(sampleAtts))
         )
+        testEntityDataSet("sample", sampleList)
       }
-      val sampleList = List(
-        EntityWithType("sample_01", "sample", Some(sampleAtts)),
-        EntityWithType("sample_02", "sample", Some(sampleAtts)),
-        EntityWithType("sample_03", "sample", Some(sampleAtts)),
-        EntityWithType("sample_04", "sample", Some(sampleAtts))
-      )
-      testDataSet("sample", sampleList)
-    }
 
-    "should work with pair data" in {
-      val pairAtts = {
-        Map(
-          "case_sample_id" -> """{"entityType": "sample", "entityName": "HCC1143"}""".parseJson,
-          "control_sample_id" -> """{"entityType": "sample", "entityName": "HCC1143_BL"}""".parseJson,
-          "participant_id" -> """{"entityType":"participant","entityName":"subject_HCC1143"}""".parseJson,
-          "header_1" -> MockUtils.randomAlpha().toJson,
-          "header_2" -> MockUtils.randomAlpha().toJson
-        )
+      "Set Data" in {
+        val samples = List(Entity(entityType = Some("sample"), entityName = Some("sample_01")),
+          Entity(entityType = Some("sample"), entityName = Some("sample_02")),
+          Entity(entityType = Some("sample"), entityName = Some("sample_03")),
+          Entity(entityType = Some("sample"), entityName = Some("sample_04")))
+        val sampleSetAtts = {
+          Map("samples" -> samples.toJson)
+        }
+        val sampleSetList = List(EntityWithType("sample_set_1", "sample_set", Some(sampleSetAtts)))
+        testMembershipDataSet("sample_set", sampleSetList, samples.size)
       }
-      val pairList = List(EntityWithType("pair_01", "pair", Some(pairAtts)))
-      testDataSet("pair", pairList)
+
     }
 
-    "should work with participant_set data" in {
-      val participantSetAtts = {
-        Map(
-          "disease" -> "lung cancer".toJson,
-          "case_sample_id" -> """{"entityType": "sample", "entityName": "HCC1143"}""".parseJson,
-          "control_sample_id" -> """{"entityType": "sample", "entityName": "HCC1143_BL"}""".parseJson,
-          "participant_id" -> """{"entityType":"participant_set","entityName":"subject_HCC1143"}""".parseJson,
-          "gender" -> "male".toJson,
-          "header_1" -> MockUtils.randomAlpha().toJson,
-          "header_2" -> MockUtils.randomAlpha().toJson
-        )
+    "Participant tests should pass for" - {
+
+      "Entity Data" in {
+        val participantSetAtts = {
```

Comment:
This should represent attributes for a single participant, but is named like a participant set, and contains attributes appropriate for pairs.